src/hotspot/os/linux/perfMemory_linux.hpp

```diff
@@ -30,10 +30,6 @@
 class PerfMemoryLinux : AllStatic {
 
 public:
-  static inline const char* perfdata_name() {
-    return "perfdata";
-  }
-
   static bool checkpoint(const char* checkpoint_path);
```

Comment:
With the proposed changes you do not need the `checkpoint_path` parameter anymore.

Reply:
Thanks for noting that, updated.